src/test/java/org/apache/commons/text/TextStringBuilderTest.java

```diff
@@ -1057,15 +1057,24 @@ public void testHashCode() {
         final TextStringBuilder sb = new TextStringBuilder();
         final int hc1a = sb.hashCode();
         final int hc1b = sb.hashCode();
-        final int emptyHc = Arrays.hashCode(sb.getBuffer());
-        assertEquals(emptyHc, hc1a);
```

Comment:
I'm not a fan of removing passing asserts because it could hide a regression.

Reply:
The point here is that this assert is not passing anymore as hashcode of the buffer is not equals to the hashcode of the TextStringBuilder itself

Reply:
@sebx59 maybe instead of removing the asserts (this :point_up: and :point_down: ) we could invert them then, if the behavior has changed?

Reply:
you're right, I checkded that the 2 hashcodes are not equals